Go/ds/inorder_traversal_binarytree.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import "fmt"
+
+//Node defines the structure of a particular node in binary tree
+type Node struct {
+	data  int
+	left  *Node
+	right *Node
+}
+
+//NewNode creates a node and returns it's pointer
+func NewNode(data int) *Node {
+	return &Node{
+		data: data,
+	}
+}
+
+//InorderTraversal prints the Inorder traversal of tree
+func (root *Node) InorderTraversal() {
+	//Inorder Traversal : Left -> Root -> Right
+	if root != nil {
+		root.left.InorderTraversal()
+		fmt.Printf("%v ", root.data)
+		root.right.InorderTraversal()
+
+	}
+}
+
+//Driver Code
+func main() {
+	//Creating a Tree with following structure
+	/*
+			5
+		   / \
+		  6   7
+		 /   / \
+		3   10  9
+	*/
+	root := NewNode(5)
```

Comment:
Can you please take the node value input from user @Kathan-Vakharia ?

Reply:
@Kajol-Kumari I have implemented your changes, Can you merge it now  :)